### ISSUE TITLE
COLA: COarse LAbel pre-training for 3D semantic segmentation of sparse LiDAR datasets. (arXiv:2202.06884v3 [cs.CV] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#25260]
COLA: COarse LAbel pre-training for 3D semantic segmentation of sparse LiDAR datasets. (arXiv:2202.06884v3 [cs.CV] UPDATED)<br>
https://ift.tt/oVkjMtS<br>
<p>Transfer learning is a proven technique in 2D computer vision to leverage the large amount of data available and achieve high performance with datasets limited in size due to the cost of acquisition or annotation. In 3D, annotation is known to be a costly task; nevertheless, pre-training methods have only recently been investigated. Due to this cost, unsupervised pre-training has been heavily favored. In this work, we tackle the case of real-time 3D semantic segmentation of sparse autonomous driving LiDAR scans. Such datasets have been increasingly released, but each has a unique label set. We propose here an intermediate-level label set called coarse labels, which can easily be used on any existing and future autonomous driving datasets, thus allowing all the data available to be leveraged at once without any additional manual labeling. This way, we have access to a larger dataset, alongside a simple task of semantic segmentation. With it, we introduce a new pre-training task: coarse label pre-training, also called COLA. We thoroughly analyze the impact of COLA on various datasets and architectures and show that it yields a noticeable performance improvement, especially when only a small dataset is available for the finetuning task.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
